dashboard/test/helpers/script_levels_helper_test.rb

```diff
@@ -25,20 +25,20 @@ class ScriptLevelsHelperTest < ActionView::TestCase
   test 'script name instead of stage name in header for HOC' do
     self.stubs(:current_user).returns(nil)
     script_level = Script.find_by_name(Script::HOC_NAME).get_script_level_by_chapter 1
-    assert_equal 'Hour of Code', script_level.stage.summarize[:title]
+    assert_equal 'Hour of Code', header_progress(script_level)[:title]
   end
 
   test 'show stage name in header for multi-stage script' do
     self.stubs(:current_user).returns(nil)
     script = Script.find_by_name(Script::COURSE4_NAME)
     script_level = script.get_script_level_by_stage_and_position 3, 1
-    assert_equal 'Stage 3: ' + I18n.t("data.script.name.#{script.name}.#{script_level.stage.name}"), script_level.stage.summarize[:title]
+    assert_equal 'Stage 3: ' + I18n.t("data.script.name.#{script.name}.#{script_level.stage.name}"), header_progress(script_level)[:title]
   end
 
   test 'show stage position in header for default script' do
     self.stubs(:current_user).returns(nil)
     script_level = Script.twenty_hour_script.script_levels.fifth
-    assert_equal 'Stage 2: The Maze', script_level.stage.summarize[:title]
+    assert_equal 'Stage 2: The Maze', header_progress(script_level)[:title]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.